LeagueOfLegends/ChampionModules/TwistedFateModule.cs

```diff
@@ -17,29 +15,10 @@ class TwistedFateModule : ChampionModule
         HSVColor RColor = new HSVColor(0.81f, 0.43f, 1);
         HSVColor RColor2 = new HSVColor(0.91f, 0.87f, 1);
 
-        /// <summary>
-        /// Creates a new champion instance.
-        /// </summary>
-        /// <param name="ledCount">Number of LEDs in the strip</param>
-        /// <param name="gameState">Game state data</param>
-        /// <param name="preferredLightMode">Preferred light mode</param>
-        /// <param name="preferredCastMode">Preferred ability cast mode (Normal, Quick Cast, Quick Cast with Indicator)</param>
-        public static TwistedFateModule Create(int ledCount, GameState gameState, LightingMode preferredLightMode, AbilityCastPreference preferredCastMode = AbilityCastPreference.Normal)
-        {
-            return new TwistedFateModule(ledCount, gameState, CHAMPION_NAME, preferredLightMode, preferredCastMode);
-        }
-
-
-        private TwistedFateModule(int ledCount, GameState gameState, string championName, LightingMode preferredLightMode, AbilityCastPreference preferredCastMode)
-            : base(ledCount, championName, gameState, preferredLightMode, preferredCastMode)
+        public TwistedFateModule(int ledCount, GameState gameState, LightingMode preferredLightMode, AbilityCastPreference preferredCastMode)
```

Comment:
I agree about removing the factory methods, but please keep the parameter documentation for now, so the modules as as self-explanatory as possible in case anyone wants to create their own controller.

Reply:
Like, the documentation the factory methods had? I haven't seen any useful documentation anywhere else in the code though. The only useful documentation would be for the abstract methods, which are a necessity to implement in every module.

Reply:
Yes, I should document all of the abstract methods. Don't bother, though, because I'm going to be touching that for the lighting system rework

Reply:
So then this PR can be considered complete without the documentation?